apps/web/ui/search/degree/degree-picker.tsx

```diff
@@ -70,7 +55,7 @@ export function DegreePicker(props: {
         <p>
           The following remaining degree modules will be placed in the graph:
         </p>
-        <p className="mb-0">{remainingCodes.join(', ')}</p>
+        <p className="mb-0">{buildList.join(', ')}</p>
```

Comment:
This is meant to exclude user modules done and modules doing, as they are mentioned in a previous section on this page